src/main/java/com/staffmanagement/controller/UserCrudController.java

```diff
@@ -0,0 +1,67 @@
+package com.staffmanagement.controller;
```

Comment:
create another package `v1` in `controller`. Versioning of controller and service classes is important.